src/oop/user_defined_classes/hard/q6/UniversityStudent.java

```diff
@@ -0,0 +1,19 @@
+package oop.user_defined_classes.hard.q6;
+
+public class UniversityStudent {
+    private int fatigueLevel;
+    private int workload;
+
+    public UniversityStudent(int level, int work) {
+        fatigueLevel = level;
+        workload = work;
+    }
+
+    public void workAllDay() {
+        while (workload > 0) {
+            fatigueLevel++;
+            workload--;
+        }
+        System.out.println("Fatigue level is " + fatigueLevel);
```

Comment:
We can maybe print the workload and check if the workload is more than zero

Reply:
I don't see the purpose in printing the workload since it should always be 0 but I will add an assert to check the field